Site/dataobjects/SiteVideoMedia.php

```diff
@@ -258,6 +275,55 @@ public function getScrubberImageWidth()
 	}
 
 	// }}}
+	// {{{ public function getFileDirectory()
+
+	public function getFileDirectory($encoding_shortname)
+	{
+		$items = array(
+			$this->getFileBase(),
+			$this->id,
+			'full'
+		);
+
+		return implode(DIRECTORY_SEPARATOR, $items);
+	}
+
+	// }}}
+	// {{{ public function getFilename()
+
+	public function getFilename($encoding_shortname)
+	{
+		$binding = $this->getEncodingBinding($encoding_shortname);
+
+		if ($this->getMediaSet()->obfuscate_filename) {
+			$filename = $this->filename;
+		} else {
+			$filename = $this->id;
+		}
```

Comment:
The `$filename` variable isn't being used anywhere. We should use it in the `sprintf` father down and replace line 301 with `$filename = $encoding_shortname`.